automation/DeployBrowsers/configure_firefox.py

```diff
@@ -86,6 +86,11 @@ def privacy(browser_params, fp, fo, root_dir, browser_profile_path):
         shutil.copy(os.path.join(root_dir, 'firefox_extensions',
                                  'ublock_origin', 'storage.js'), ublock_dir)
 
+    if browser_params['no-popups']:
+        # Disable popups
+        fo.set_preference("browser.link.open_newwindow.restriction", 0)
```

Comment:
Mind to give a brief description of these two prefs? Even just a copy/paste from http://kb.mozillazine.org/Browser.link.open_newwindow and http://kb.mozillazine.org/Browser.link.open_newwindow.restriction?